bin/blin.p6

```diff
@@ -51,6 +58,14 @@ my $skip-tests = (
    # These seem to hang and leave some processes behind:
    ‘IO::Socket::Async::SSL’,
    ‘IRC::Client’,
+   ‘Perl6::Ecosystem’,           # eats memory
+   ‘Concurrent::BoundedChannel’, # hangs
+   ‘Net::FTP’,                   # hangs
+   ‘Syslog::Parse’,              # hangs
+   ‘Fcntl’,                      # hangs
+   ‘Selenium::WebDriver’,        # hangs
+   ‘WebService::Soundcloud’,     # hangs
+   ‘UNIX::Daemonize’,            # hangs
```

Comment:
Are those hangs Pakku specific? I can install most of them (except Perl6::Ecosystem, see https://github.com/JJ/raku-river/issues/16) just fine.

Reply:
I'll get back to you on these modules but for now, I just realized that `Fcntl` is Pakku specific.

When building Fcntl it hangs waiting for output of the `Build` stage [here](https://github.com/hythm7/Pakku/blob/main/lib/Pakku/Builder.rakumod#L47), I haven't seen this issue with other modules.

looks like this `shell` command [here](https://github.com/manchicken/perl6-Fcntl/blob/main/Build.pm#L17) suppresses the output somehow, and if I changed this line to `shell( %vars<MAKE>, :err );` it works fine and doesn't hang anymore, `Pakku` receives the full output of the `Build` stage. I don't understand why, but still looking. (Tested that by cloning `Fcntl` and changed the said line then ran `pakku verbose trace build ./Fcntl`)

I'm also looking into a different issue I just noticed, so will close the PR and will re-open when it's ready.

Reply:
Perl6::Ecosystem works perfectly now. I haven't released a new version, but it does not hang when installed from HEAD.

Reply:
Thanks @JJ, Confirmed. I did not see the hang issue when tested the module from HEAD. and since the name of the module on HEAD is now `Raku::Ecosystem` not `Perl6::Ecosystem`, I left `Perl6::Ecosystem` in the list to avoid testing the old version (in case anyone used the old name `Perl6::Ecosystem`).

Reply:
@Altai-man Only `Fcntl` was `Pakku` specific [issue](https://gist.github.com/hythm7/7bafa02262ef1484ce3e39dd7472c2f1), and has been fixed with the changes to `Builder.rakumod` file in this [commit](https://github.com/hythm7/Pakku/commit/9d53457458b733f4ee350d65b33547f75527372c).

The remaining modules show same behavior using `Pakku` and `Zef` (some tests hang then get killed after timeout), so removing them from the list.

I did not know how to re-open this PR with the new changes, so I opend a new one [here](https://github.com/Raku/Blin/pull/26), Please let me know if anything needed.